core-java/src/main/java/com/baeldung/utils/Utils.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.utils;
+
+import java.util.Arrays;
+
+public class Utils {
+
+        public static void main(String[] args) {
+                String testString = "abcdefg";
+                char[] testArray = { 'e', 'g', 'p' };
+
+                System.out.println(testString + " to array => " + Arrays.toString(stringToCharArray(testString)));
+                System.out.println(Arrays.toString(testArray) + " to string => " + charArrayToString(testArray));
+        }
+
+        public static char[] stringToCharArray(final String input) {
+                if (null == input)
+                        return null;
+
+                return input.toCharArray();
+        }
+
+        public static String charArrayToString(final char[] charArray) {
+                if (null == charArray)
```

Comment:
The formatting here does not follow the convention. Keep in mind that you have a predefined formatters for Eclipse and IDEA in the repo